Fix benchmark scripts (#1517)
opea-project/GenAIExamples: README-deploy-benchmark.md

   - Kubernetes installation: Use [kubespray](https://github.com/opea-project/docs/blob/main/guide/installation/k8s_install/k8s_install_kubespray.md) or other official Kubernetes installation guides
   - (Optional) [Kubernetes set up guide on Intel Gaudi product](https://github.com/opea-project/GenAIInfra/blob/main/README.md#setup-kubernetes-cluster)

2. **Configuration YAML**  

>    The configuration file (e.g., `./ChatQnA/benchmark_chatqna.yaml`) consists of two main sections: deployment and benchmarking. Required fields with `# mandatory` comment must be filled with valid values, such as `HUGGINGFACEHUB_API_TOKEN`. For all other fields, you can either customize them according to their needs or leave them empty ("") to use the default values from the [helm charts](https://github.com/opea-project/GenAIInfra/tree/main/helm-charts).

Reviewer: ```suggestion
   The configuration file (e.g., `./ChatQnA/benchmark_chatqna.yaml`) consists of two main sections: deployment and benchmarking. Required fields with `# mandatory` comment must be filled with valid values, such as `HUGGINGFACEHUB_API_TOKEN`. For all other fields, you can either customize them according to our needs or leave them empty ("") to use the default values from the [helm charts](https://github.com/opea-project/GenAIInfra/tree/main/helm-charts).
```

   **Default Models**:

   - LLM: `meta-llama/Meta-Llama-3-8B-Instruct` (Required: must be specified as it's shared between deployment and benchmarking phases)
   - Embedding: `BAAI/bge-base-en-v1.5`
   - Reranking: `BAAI/bge-reranker-base`

>    You can customize these models by setting the `model_id` field in the corresponding service section. Note that the LLM model must be specified in the configuration as it is used by both deployment and benchmarking processes.

Reviewer: ```suggestion
   You can customize which models to use by setting the `model_id` field in the corresponding service section. Note that the LLM model must be specified in the configuration as it is used by both deployment and benchmarking processes.
```

   **Important Notes**:

   - For Gaudi deployments:
     - LLM service runs on Gaudi devices
     - If enabled, the reranking service (teirerank) also runs on Gaudi devices
   - **Llama Model Access**:
     - Downloading Llama models requires both:
       1. HuggingFace API token
       2. Special authorization from Meta
     - Please visit [meta-llama/Meta-Llama-3-8B-Instruct](https://huggingface.co/meta-llama/Meta-Llama-3-8B-Instruct) to request access
     - Deployment will fail if model download is unsuccessful due to missing authorization

   **Node and Replica Configuration**:

   ```yaml
   node: [1, 2, 4, 8] # Number of nodes to deploy
   replicaCount: [1, 2, 4, 8] # Must align with node configuration
   ```

   The `replicaCount` values must align with the `node` configuration by index:

   - When deploying on 1 node → uses replicaCount[0] = 1
   - When deploying on 2 nodes → uses replicaCount[1] = 2
   - When deploying on 4 nodes → uses replicaCount[2] = 4
   - When deploying on 8 nodes → uses replicaCount[3] = 8

>    Note: Model parameters that accept lists (e.g., `max_batch_size`, `max_num_seqs`) are deployment parameters that affect model behavior but not service count. When these parameters are lists, each value will trigger a service upgrade followed by a new round of testing, while maintaining the same number of service instances.

Reviewer: ```suggestion
   Note: Model parameters that accept lists (e.g., `max_batch_size`, `max_num_seqs`) are deployment parameters that affect model service behavior but not the number of service instances. When these parameters are lists, each value will trigger a service upgrade followed by a new round of testing, while maintaining the same number of service instances.
```
